Doc/library/lzma.rst

```diff
@@ -147,7 +147,7 @@ Compressing and decompressing data in memory
       This format is more limited than ``.xz`` -- it does not support integrity
       checks or multiple filters.
 
-   * :const:`FORMAT_RAW`: A raw data stream, not using sequences format.
+   * :const:`FORMAT_RAW`: A raw data stream, not using any container format.
```

Comment:
I don't know what the motivation is for this change, and it doesn't seem related to the linked issue. Why is it needed?

Reply:
It was (seemingly) mistakenly changed in https://github.com/python/cpython/pull/92450/files , a previous attempt to fix the issue